src/components/FormField/commonElement.tsx

```diff
@@ -33,3 +33,10 @@ export const EmptyButton = styled.button`
     cursor: not-allowed;
   }
 `;
+
+export const GridCenter = styled.div`
```

Comment:
Whats this doing? Is it worth trying to accomplish it in flex so we're not mixing flex and grid unnecessarily?

Reply:
This is not needed everywhere
Just in options that need content to be displayed in multiple sides
This would be removed in my upcoming FlexContainer Pr that I'm currently working on